### PR TITLE
2 CLB Cards

### LINKED ISSUE #959: Young Blue Dragon // Sand Augury

### LINKED ISSUE #960: Young Red Dragon // Bathe in Gold

### PR DESCRIPTION
**Pull request by JohnWilliams77**
-closes #959 
-closes #960

### CODE REVIEW

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/young_blue_dragon.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+AlternateMode:Adventure
+
+ALTERNATE
+
+Name:Sand Augury
+ManaCost:1 U
+Types:Sorcery Adventure
+A:SP$ Scry | ScryNum$ 1 | SubAbility$ DBDraw | SpellDescription$ Scry 1,
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ 1 | SpellDescription$ then draw a card
+Oracle:Scry 1, then draw a card. (Then exile this card. You may cast the creature later from exile.)
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/young_blue_dragon.txt:16]: File name should be: young_blue_dragon_sand_augury.txt

File: forge-gui/res/cardsfolder/upcoming/young_red_dragon.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+AlternateMode:Adventure
+
+ALTERNATE
+
+Name:Bathe in Gold
+ManaCost:1 R
+Types:Instant Adventure
+A:SP$ Token | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token. (Then exile this card. You may cast the creature later from exile.)
+DeckHas:Ability$Token
+Oracle:Create a Treasure token. (Then exile this card. You may cast the creature later from exile.)
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/young_red_dragon.txt:17]: File name should be: young_red_dragon_bathe_in_gold.txt

File: forge-gui/res/cardsfolder/upcoming/young_red_dragon.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Oracle:Flying\nYoung Red Dragon can’t block.
+AlternateMode:Adventure
+
+ALTERNATE
+
+Name:Bathe in Gold
+ManaCost:1 R
+Types:Instant Adventure
+A:SP$ Token | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token. (Then exile this card. You may cast the creature later from exile.)
+DeckHas:Ability$Token
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/young_red_dragon.txt:16]: DeckHas:Ability$Token|Sacrifice & Type$Artifact|Treasure

**Comment by paulsnoops** [comment]
Looks good! @JohnWilliams77 I invited you to join Forge contributors. If you accept you should be able to assign yourself to card scripts on one of the projects here: https://github.com/Card-Forge/forge/projects?type=beta

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**